tests/ScopedNodeTest.php

```diff
@@ -219,4 +221,12 @@ public function testInsertingBeforeAnotherScopeFails()
 
 		$a->insertAfterNode($b);
 	}
+
+	// public function testEagerLoadingAncestorsWithScope()
+	// {
+	// 	$filteredNodes = MenuItem::where('title', 'menu item 3')->with(['ancestors'])->get();
+
+	// 	$this->assertEquals(2, $filteredNodes->find(5)->ancestors[0]->id);
+	// 	$this->assertEquals(4, $filteredNodes->find(6)->ancestors[0]->id);
+	// }
```

Comment:
Not quite ready?

Reply:
That was from the original repo.
But I didn't really dig into it as I cherry picked commits.